examples/polars/notebook.ipynb

```diff
@@ -193,34 +408,28 @@
     "# To visualize do `pip install \"sf-hamilton[visualization]\"` if you want these to work\n",
     "dr.visualize_execution(output_columns, './polars', {\"format\": \"png\"})\n",
     "dr.display_all_functions('./my_full_dag.dot')"
```

Comment:
oh in a notebook this can be simplified to:
```suggestion
    "dr.visualize_execution(output_columns)\n",
    "dr.display_all_functions()"
```
though I think only one will display in jupyter at a time.

Reply:
thanks for the heads up